docs/api_specification.md

```diff
@@ -0,0 +1,213 @@
+# API Specification
+
+## POST /v1/api/products
+
+This endpoint creates a new product in pay-apps.
+
+### Request example
+
+```
+POST /v1/api/products
+Authorization: Bearer API_TOKEN
+Content-Type: application/json
+
+{
+    "external_service_id" :  "d76tt83563thcert8urihgiuer",
+    "pay_api_token" :        "5meusgv5ufclsbvde78mdf35bfdhnhm1307euq94kcf0lcqcqrovbjovib",
```

Comment:
@kakumara minor point for future is that I would avoid using "sane looking" keys in the code base as they stand out and to reviewers later in the day could cause alarm '*5meusgv5ufclsbv**EXAMPLE**qcqrovbjovib*' may be enough